airbyte-integrations/connectors/source-chargebee/source_chargebee/schemas/item_price.json

```diff
@@ -0,0 +1,205 @@
+      "maxLength": 100
+    },
+    "item_id": {
+      "type": ["string", "null"],
+      "maxLength": 100
+    },
+    "description": {
+      "type": ["string", "null"],
+      "maxLength": 500
+    },
+    "status": {
+      "type": ["string", "null"],
+      "enum": ["active", "archived", "deleted"]
+    },
+    "external_name": {
+      "type": ["string", "null"],
+      "maxLength": 100
+    },
+    "pricing_model": {
+      "type": ["string", "null"],
+      "enum": ["flat_fee", "per_unit", "tiered", "volume", "stairstep"]
+    },
+    "price": {
+      "type": ["integer", "null"],
+      "minimum": 0
+    },
+    "price_in_decimal": {
+      "type": ["string", "null"],
+      "maxLength": 33
+    },
+    "period": {
+      "type": ["integer", "null"],
+      "minimum": 1
+    },
+    "currency_code": {
+      "type": ["string", "null"],
+      "maxLength": 3
+    },
+    "period_unit": {
+      "type": ["string", "null"],
+      "enum": ["day", "week", "month", "year"]
+    },
+    "trial_period": {
+      "type": ["integer", "null"],
+      "minimum": 1
+    },
+    "trial_period_unit": {
+      "type": ["string", "null"],
+      "enum": ["day", "month"]
+    },
+    "trial_end_action": {
+      "type": ["string", "null"],
+      "enum": ["site_default", "activate_subscription", "cancel_subscription"]
+    },
+    "shipping_period": {
+      "type": ["integer", "null"],
+      "minimum": 1
+    },
+    "shipping_period_unit": {
+      "type": ["string", "null"],
+      "enum": ["day", "week", "month", "year"]
+    },
+    "billing_cycles": {
+      "type": ["integer", "null"],
+      "minimum": 1
+    },
+    "free_quantity": {
+      "type": ["integer", "null"],
+      "minimum": 0
+    },
+    "free_quantity_in_decimal": {
+      "type": ["string", "null"],
+      "maxLength": 33
+    },
+    "resource_version": {
+      "type": ["integer", "null"]
+    },
+    "updated_at": {
+      "type": ["integer", "null"]
+    },
+    "created_at": {
+      "type": ["integer", "null"]
+    },
+    "archived_at": {
+      "type": ["integer", "null"]
+    },
+    "invoice_notes": {
+      "type": ["string", "null"],
+      "maxLength": 2000
+    },
+    "is_taxable": {
+      "type": ["boolean", "null"]
+    },
+    "metadata": {
+      "type": ["object", "null"],
+      "properties": {}
+    },
+    "item_type": {
+      "type": ["string", "null"],
+      "enum": ["plan", "addon", "charge"]
+    },
+    "show_description_in_invoices": {
+      "type": ["boolean", "null"]
+    },
+    "show_description_in_quotes": {
+      "type": ["boolean", "null"]
+    },
+    "tiers": {
+      "type": ["array", "null"],
+      "items": {
+        "type": ["object", "null"],
+        "properties": {
+          "starting_unit": {
+            "type": ["integer", "null"],
+            "minimum": 1
+          },
+          "ending_unit": {
+            "type": ["integer", "null"]
+          },
+          "price": {
+            "type": ["integer", "null"],
```

Comment:
Could price has non integer value? I would recommend not using integer type in json schema at all,  and use "number" instead unless we have some reasons to do it.

Reply:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/19872253/129515933-ae3e509b-459f-44f6-91f8-b4a4007e6588.png">

Reply:
Btw, why you don't recommend using integer type?

Reply:
Cause jsonschema's number type is universal and works either for integers and float.

Reply:
But why do we need a float if we are sure that type is integer?